src/components/Login/index.js

```diff
@@ -41,15 +41,15 @@ function HeaderBar() {
         };
     }, [prevScrollY]);
 
-    const headerClasses = `fixed w-full top-0 h-14 bg-white text-gray-700 z-40  border-b border-gray-200 ${
```

Comment:
This change was breaking the structure. video z index was overpowering almost everything including the upload new post button and sidebar menu. Have fixed it in #98